doc/source/cluster/cloud.rst

```diff
@@ -272,6 +272,116 @@ There are two ways of running private clusters:
             $ ray down ray/python/ray/autoscaler/local/example-full.yaml
 
 
+.. _manual-cluster:
+
+Manual Ray Cluster Setup
+------------------------
+
+The most preferable way to run a Ray cluster is via the Ray Cluster Launcher. However, it is also possible to start a Ray cluster by hand.
+
+This section assumes that you have a list of machines and that the nodes in the cluster can communicate with each other. It also assumes that Ray is installed
+on each machine. To install Ray, follow the `installation instructions`_.
+
+.. _`installation instructions`: http://docs.ray.io/en/master/installation.html
+
+Starting Ray on each machine
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the head node (just choose some node to be the head node), run the following.
+If the ``--port`` argument is omitted, Ray will choose port 6379, falling back to a
+random port.
+
+.. code-block:: bash
+
+  $ ray start --head --port=6379
+  ...
+  Next steps
+    To connect to this Ray runtime from another node, run
+      ray start --address='<ip address>:6379' --redis-password='<password>'
+
+  If connection fails, check your firewall settings and network configuration.
+
+The command will print out the address of the Redis server that was started
+(the local node IP address plus the port number you specified).
+
+**Then on each of the other nodes**, run the following. Make sure to replace
+``<address>`` with the value printed by the command on the head node (it
+should look something like ``123.45.67.89:6379``).
+
+Note that if your compute nodes are on their own subnetwork with Network
+Address Translation, to connect from a regular machine outside that subnetwork,
+the command printed by the head node will not work. You need to find the
+address that will reach the head node from the second machine. If the head node
+has a domain address like compute04.berkeley.edu, you can simply use that in
+place of an IP address and rely on the DNS.
+
+.. code-block:: bash
+
+  $ ray start --address=<address> --redis-password='<password>'
+  --------------------
+  Ray runtime started.
+  --------------------
+
+  To terminate the Ray runtime, run
+    ray stop
+
+If you wish to specify that a machine has 10 CPUs and 1 GPU, you can do this
+with the flags ``--num-cpus=10`` and ``--num-gpus=1``. See the :ref:`Configuration <configuring-ray>` page for more information.
```

Comment:
This detection of resources happens automatically tho, yes? Just don't want people thinking they need to do this. 

Could add "but Ray will automatically detect CPU and GPU resources, so there is no need to do this unless you want to limit the resources availiable to Ray" or something